BlueprintUI/Sources/Internal/RoundingCorrectionKey.swift

```diff
@@ -0,0 +1,13 @@
+import CoreGraphics
+
+/// This key is used internally to cascade rounding information to nested BlueprintViews.
+enum RoundingCorrectionKey: EnvironmentKey {
+    static let defaultValue: CGRect = .zero
+}
+
+extension Environment {
```

Comment:
Should we stick this on the env, or should we change the inherited (thing) type to something other than env?

Reply:
(Eg, change this https://github.com/square/Blueprint/blob/main/BlueprintUI/Sources/Environment/Environment.swift#L84 to an `InheritedProperties` or something)

Reply:
@watt would this be preferable?

It does seem like it might be nice to share the `inherited` infra that already exists—it's a pattern that's being used to bridge the same space that this PR is bridging (I think?).

Reply:
I don't have a strong preference. I just put it on the environment out of convenience, since it already existed. Splitting out would require duplicating some infra, and it doesn't seem like there's much advantage.